package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "card-types",
+  "version": "1.0.0",
+  "description": "Structure of a repo:",
+  "main": "index.js",
+  "scripts": {
+    "precommit": "lint-staged",
+    "lint:eslint": "eslint --quiet .",
+    "lint:json5": "json5 -v ./**/*.json5",
+    "lint:md": "prettier --list-different ./**/*.md",
```

Comment:
Could you say, what is the role of this script if we dont use `markdown` now?

Reply:
It will be fore card-type description (there is a mention for `README.md` file )